test/fixtures/active_record.rb

```diff
@@ -1674,7 +1674,7 @@ class PainterResource < JSONAPI::Resource
       has_many :paintings
 
       filter :name, apply: lambda { |records, value, options|
-        records.where('name LIKE ?', value)
+        records.where("#{_model_class.table_name}.name LIKE ?", value)
```

Comment:
I think this has become standard practice for complex filters that may be referenced in an included object, but happy to write this a better way if one exists.